### PR TITLE
SDLC Checklists

### PR DESCRIPTION
**Pull request by tamigoodall**
Created checklists.md file (from B2C docs branch)

### CODE REVIEW

File: articles/architecture-scenarios/checklists.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - checklists
+    - best practices
+    - implementation checklist
+contentType: reference
+useCase:
+    - implementation
+---
+# Implementation Checklists
+
+Auth0 recommends that you break your implementation into the following 6 phases to plan and complete your implementation.
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md]:
```suggestion
Auth0 provides a number of Checklist (in Google Sheets format) that will help you co-ordinate progress as you work through each phase in your Software Development Life Cycle (SDLC).

For each SDLC phase, the relevant Checklist (see below) will provided you with a list of items you will need to consider, together with links to pertinent Auth0 feature documentation covering each. Not all items will necessary apply to the project you are working on, so we advise you to copy and modify a checklist to suit your needs.  
```

File: articles/architecture-scenarios/checklists.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Auth0 recommends that you break your implementation into the following 6 phases to plan and complete your implementation. 
+
+* [Analyze](#analyze)
+* [Design](#design)
+* [Build](#build)
+* [Test](#test)
+* [Deploy](#deploy) 
+* [Monitor](#monitor)
+
+Auth0 provides a checklist (Google Sheets format) for each phase with links to pertinent Auth0 feature documentation. Not all parts of each phase may apply to you. You can copy and modify the checklists to suit your needs.
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* [Build](#build)
+* [Test](#test)
+* [Deploy](#deploy) 
+* [Monitor](#monitor)
+
+Auth0 provides a checklist (Google Sheets format) for each phase with links to pertinent Auth0 feature documentation. Not all parts of each phase may apply to you. You can copy and modify the checklists to suit your needs. 
+
+## Analyze
+
+Use the [Analyze checklist](https://docs.google.com/spreadsheets/d/1hTtRvvO6Szlvcr4XnsgzzmJj7FmmucAZAmFF275hB20/edit?usp=sharing) to analyze end-user business requirements and determine project goals as part of the high-level plan for the project. Convert the requirements and goals into system functions that the organization intends to develop. Activities include:
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md]:
```suggestion
Use the [Analyze checklist](https://docs.google.com/spreadsheets/d/1hTtRvvO6Szlvcr4XnsgzzmJj7FmmucAZAmFF275hB20/edit?usp=sharing) during the Analysis phase of your SDLC, where you'll analyze end-user business requirements and determine project goals as part of the high-level plan for the project. The Checklist for this phase will help you focus on Auth0 specifics as you convert the requirements and goals into system functions that you intend to develop.
```

File: articles/architecture-scenarios/checklists.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+* Gathering business requirements
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:38]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+* Creating process diagrams
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:39]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+* Performing detailed analysis
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:40]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+* Alignment to project plan
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:41]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+## Design
+
+Use the [Design checklist](https://docs.google.com/spreadsheets/d/1L0PLqcghVvCSdW_OmLoKlU0uDJtb5mw6uIufMrKW7ro/edit?usp=sharing) to describe the desired features and operations of the system, including  business rules, pseudo-code, screen layouts, and other necessary documentation. Activities include:
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md]:
```suggestion
Use the [Design checklist](https://docs.google.com/spreadsheets/d/1L0PLqcghVvCSdW_OmLoKlU0uDJtb5mw6uIufMrKW7ro/edit?usp=sharing) during the Design phase of the SDLC, to help you focus on Auth0 specifics as you describe the desired features and operations of the system, including business rules, pseudo-code, screen layouts, and any other necessary collateral.
```

File: articles/architecture-scenarios/checklists.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+* Infrastructure design
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:49]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+* System model design
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:50]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+## Build
+
+Use the [Build checklist](https://docs.google.com/spreadsheets/d/1Wd5HwONEi2Max7zcORJpwn8yQO_mbu62FDc-Sk8oWS8/edit?usp=sharing) to develop the actual system through implementation of infrastructure and code. Activities include:
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md]:
```suggestion
Use the [Build checklist](https://docs.google.com/spreadsheets/d/1Wd5HwONEi2Max7zcORJpwn8yQO_mbu62FDc-Sk8oWS8/edit?usp=sharing) during the Build phase of the SDLC, to help you focus on Auth0 specifics as you develop the actual system through implementation of infrastructure and code.
```

File: articles/architecture-scenarios/checklists.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+* Infrastructure implementation
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:58]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+* Code implementation
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:59]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+
+## Test
+
+Use the [Test checklist](https://docs.google.com/spreadsheets/d/19tQOQ_heBVpHXx-YKZZY20u6y8mU8IwRHEF32znZ_-w/edit?usp=sharing) to integrate and deploy all implemented code in the testing environment infrastructure. Testing then follows Software Testing Life Cycle activities to check the system for errors, bugs, and defects to verify that system functionalities work as expected (or not). Activities include:
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md]:
```suggestion
Use the [Test checklist](https://docs.google.com/spreadsheets/d/19tQOQ_heBVpHXx-YKZZY20u6y8mU8IwRHEF32znZ_-w/edit?usp=sharing) during the Test phase of the SDLC, to help focus on Auth0 specifics as you integrate and deploy all implemented code in your testing environment infrastructure and perform testing in line with your Software Testing Life Cycle activities (in order to check the system for errors, bugs, and defects).
```

File: articles/architecture-scenarios/checklists.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+* Write test cases
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:67]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+* Execute test cases
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:68]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+
+## Deploy
+
+Use the [Deploy checklist](https://docs.google.com/spreadsheets/d/1MRBz4CiT6RTL7Mf7eWmpjSZEQS3rwjNryyPGmvjn-fc/edit?usp=sharing) to deploy the system to either a staging or production environment, where actual users begin to operate and interact with it.
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md]:
```suggestion
Use the [Deploy checklist](https://docs.google.com/spreadsheets/d/1MRBz4CiT6RTL7Mf7eWmpjSZEQS3rwjNryyPGmvjn-fc/edit?usp=sharing) during the Deployment phase of the SDCL, to help focus on Auth0 specifics as you prepare to deploy the system to either a staging or production environment, where actual users will begin to operate and interact with it. 
```

File: articles/architecture-scenarios/index.md [hunk 18]
```diff
@@ -16,9 +16,9 @@ useCase:
 ---
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/index.md]: Should this be part of this Pull request?

File: articles/architecture-scenarios/checklists.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+
+## Deploy
+
+Use the [Deploy checklist](https://docs.google.com/spreadsheets/d/1MRBz4CiT6RTL7Mf7eWmpjSZEQS3rwjNryyPGmvjn-fc/edit?usp=sharing) to deploy the system to either a staging or production environment, where actual users begin to operate and interact with it. 
+
+Eventually, you deploy all components of the system to the production environment as a live release is made.
+
+## Monitor
+
+Use the [Monitor checklist](https://docs.google.com/spreadsheets/d/1zHs29OoeiludHxY5cV06oGVAIW91jMjUflJgzJVaRKo/edit?usp=sharing) to make enhancements, corrections, and changes to ensure the system continues to work and stays updated to meet the business objectives and support the needs of the users. Activities include:
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md]:
```suggestion
Use the [Monitor checklist](https://docs.google.com/spreadsheets/d/1zHs29OoeiludHxY5cV06oGVAIW91jMjUflJgzJVaRKo/edit?usp=sharing) during the Monitoring phase of the SDLC, to help focus on Auth0 specifics as you make enhancements, corrections, and changes to your live production environment, in order to ensure the system continues to work as desired and stays updated to meet the business objectives as well as support the needs of the your users. 
```

File: articles/architecture-scenarios/checklists.md [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+
+Use the [Deploy checklist](https://docs.google.com/spreadsheets/d/1MRBz4CiT6RTL7Mf7eWmpjSZEQS3rwjNryyPGmvjn-fc/edit?usp=sharing) to deploy the system to either a staging or production environment, where actual users begin to operate and interact with it. 
+
+Eventually, you deploy all components of the system to the production environment as a live release is made.
+
+## Monitor
+
+Use the [Monitor checklist](https://docs.google.com/spreadsheets/d/1zHs29OoeiludHxY5cV06oGVAIW91jMjUflJgzJVaRKo/edit?usp=sharing) to make enhancements, corrections, and changes to ensure the system continues to work and stays updated to meet the business objectives and support the needs of the users. Activities include: 
+
+* Monitoring
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:84]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+* Maintenance
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:85]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+* Changes and adjustments
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:86]:
```suggestion

```

File: articles/architecture-scenarios/checklists.md [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+* Upgrade and adapt to future needs
```
Comment (PeterGFernandez) [inline articles/architecture-scenarios/checklists.md:87]:
```suggestion

```

**Review by jeffreylees — APPROVED** [review]

**Pull request merged**